src/doom/g_game.c

```diff
@@ -1319,6 +1319,13 @@ int cpars[32] =
     240,150,180,150,150,300,330,420,300,180,	// 21-30
     120,30					// 31-32
 };
+
+// Chex Quest Par Times
+int chexpars[2][6] = 
```

Comment:
I'd go for `chexpars[6]`, remove the `{0}` and use `chexpars[gamemap]` later. You already checked that `gameepisode == 1` just a few lines earlier.